faq.html

```diff
@@ -304,18 +304,11 @@ <h1>FAQs</h1>
                                                 all of the offered ROHP dates the same?</a>
                                             <div id="accordion-10"
                                                 class="accordion-section-content">
-                                                <p class="darkcolor">Yes, essentially. They will all
-                                                    follow a similar
-                                                    format and feature the same activities. However,
-                                                    these will vary for each program, depending on
-                                                    speaker/performer availability. No program will
-                                                    be
-                                                    significantly different from another so you will
-                                                    not
-                                                    be missing out! Please check out the specific
-                                                    itinerary for each date under the "Agenda" Tab
-                                                    if
-                                                    you are interested in the activities that day.
+                                                <p class="darkcolor">No. This year, we will be offering 2 types of programs: virtual and hybrid. Though most of the activities we will be doing are the same or very similar, some activities, such as the campus/focus tours, will be held in-person in groups versus online through videos. 
```

Comment:
Can you add something to the effect of "specifics on performance groups/speakers..." will be confirmed closer to the program date?

Reply:
Also clarify that the meeting w a host is one-on-one for virtual